allennlp/data/token_indexers/token_characters_indexer.py

```diff
@@ -33,10 +36,12 @@ class TokenCharactersIndexer(TokenIndexer[List[int]]):
     """
     # pylint: disable=no-self-use
     def __init__(self,
+                 min_padding_length: int = 0,
```

Comment:
This needs to be added at the end of the argument list, otherwise you'll break existing calls to the constructor (which is why tests are failing).  This is also why it needs to have a default value of 0 - we don't want to break backwards compatibility.

And, lastly, it's important for every new piece of functionality in the core library to have a test making sure it does what it's supposed to.  Can you add a test making sure that things actually get padded correctly when this parameter is used?